tests/python/gaia-ui-tests/gaiatest/apps/bzlite/regions/create_bug.py

```diff
@@ -0,0 +1,28 @@
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import Base
+
+
+class CreateBug(Base):
+
+    _title_input = (By.ID, 'summary')
+    _description_input = (By.ID, 'description')
+    _thumbnail_photo_locator = (By.CSS_SELECTOR, ".btn-file")
+    _button_submit = (By.ID, 'submit')
+
+    def tap_new_bug(self, title, description):
+        title_element = self.marionette.find_element(*self._title_input)
+        title_element.tap()
```

Comment:
I don't think you need to use .tap() here and everywhere else where you use send_keys()
That doesn't seem to happen with other usage of this function either: http://mxr.mozilla.org/gaia/search?string=send_keys
Otoh, I'm not sure if send_keys if scrolling the element into view, in that case, the tap() method might be needed, because that method includes a scroll into view method.